GRADING.md

```diff
@@ -0,0 +1,101 @@
+# Assessment Grading Guide for API Development with Java 17 & SpringBoot
+
+Welcome to the grading guide for the API Development Assessment. This guide has been meticulously designed to ensure a clear, consistent, and comprehensive evaluation of students' projects. As an assessor, you'll find a rubric below that breaks down the evaluation criteria into four distinct sections: 
+- Completion
+- Functionality
+- Code Quality & Structure
+- Language Profiency 
+
+Please go through each section methodically, ensuring that you understand the criteria and can confidently attribute a score from 0 to 5 for each student's work based on their performance in the listed areas.
+
+## Table of Contents
+
+1. [Completion](#completion)
+1. [Functionality](#functionality)
+1. [Code Quality & Structure](#code-quality--structure)
+1. [Language Proficiency](#language-proficiency)
```

Comment:
Just noticed that these are all numbered 1

Reply:
That's valid syntax for [numbered lists](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists)